### DISCUSSION TITLE
谷歌云盘为啥打不开？

### DISCUSSION THREAD

**Discussion by sunzhe1234** [AlistGo/alist#1560] [Q&A]
用软路由搭建的alist

**Comment by sunzhe1234** [comment]


**Reply by lfz-5** [reply]
图没传上来

**Reply by sunzhe1234** [reply]
> 图没传上来

传了

**Comment by sunzhe1234** [comment]
![EBEB2A65-E1F7-4837-BBF4-032454852702](https://user-images.githubusercontent.com/100278321/186401413-c83c0e45-1f3d-49c0-a5c4-34452865f370.png)
